datafusion/expr/src/expr_schema.rs

```diff
@@ -326,7 +317,12 @@ impl ExprSchemable for Expr {
     }
 }
 
-fn cast_subquery(subquery: Subquery, cast_to_type: &DataType) -> Result<Subquery> {
+/// cast subquery in InSubquery/ScalarSubquery to a given type.
```

Comment:
I find this method is used in `scalar subquer`, but the comment is for Insubquery/scalarquery.